Horatio/Scheduler/TaskScheduler.swift

```diff
@@ -61,12 +61,14 @@ class TimedTaskCoordinator : ScheduledTaskCoordinator {
     func resume() {
         isActive = true
 
-        if updateTimer == nil {
-            updateTimer = Foundation.Timer.scheduledTimer(timeInterval: Behaviors.TimerInterval, target: self, selector: #selector(timerFired), userInfo: nil, repeats: true)
-        }
-
-        if let updateTimer = updateTimer {
-            updateTimer.fire()
+        DispatchQueue.main.async {
```

Comment:
I'm not keen on forcing scheduled tasks to be run on main thread. Seems like a big hammer fix to a small hammer problem (of not dealing properly with threading throughout the app). That said, let's leave this in for now.